tests/functional/smoke/droptomb2.sh

```diff
@@ -17,7 +17,10 @@ kvs=$(kvs_create smoke-0) || $?
 
 sp='[[:space:]]'
 
-parms='kvdb-oparms csched_debug_mask=0xffffffff kvs-oparms cn_close_wait=true'
+# Set rspill params to 0x0404 (min/max of 4/4) so that we generate two
+# cn root spills for each group of eight calls to putbin.
+#
+parms='kvdb-oparms csched_rspill_params=1028 csched_debug_mask=0xffffffff kvs-oparms cn_close_wait=true'
```

Comment:
May as well fix that debug_mask while you are here? No idea why this doesn't error out.